llpc/util/llpcElfWriter.cpp

```diff
@@ -735,6 +737,282 @@ template <class Elf> void ElfWriter<Elf>::updateMetaNote(Context *pContext, cons
   pNewNote->data = data;
 }
 
+// =====================================================================================================================
+// Get the count of relocs in the reloc section
+//
+// @param relocSection : Reloc section
+template <class Elf> size_t ElfWriter<Elf>::numRelocs(const SectionBuffer *relocSection) {
+  if (relocSection->secHead.sh_size == 0)
+    return 0;
+
+  return static_cast<unsigned>(relocSection->secHead.sh_size / relocSection->secHead.sh_entsize);
+}
+
+// =====================================================================================================================
+// Get the start pos of relocs used by PS in the reloc section
+//
+// @param relocSection : Reloc section
+// @param psIsaOffset : Ps ISA offset in the elf bin
+template <class Elf> size_t ElfWriter<Elf>::getRelocPsStartPos(const SectionBuffer *relocSection, size_t psIsaOffset) {
```

Comment:
missing documentation line for `@returns`

Reply:
Fixed.